ui/push-health/Metric.jsx

```diff
@@ -25,29 +27,56 @@ export default class Metric extends React.PureComponent {
 
   render() {
     const { detailsShowing } = this.state;
-    const { result, name, value, details } = this.props;
+    const {
+      result,
+      name,
+      value,
+      details,
+      failures,
+      repo,
+      revision,
+    } = this.props;
     const resultColor = resultColorMap[result];
     const expandIcon = detailsShowing ? faMinusSquare : faPlusSquare;
 
     return (
       <td>
         <div className="d-flex flex-row">
           <div className={`bg-${resultColor} pr-2 mr-2`} />
-          <div>
-            <h3>
-              {name}
-              <span onClick={this.toggleDetails} className="btn btn-lg">
-                <FontAwesomeIcon icon={expandIcon} />
+          <div className="d-flex flex-column w-100">
```

Comment:
It might be easier to read at a glance if you use the reactstrap layout elements Row, Col, etc.

Reply:
Good call.  Done